src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    if (style.indexOf(':') === -1) {
+      return acc;
+    }
+
+    const [property, value] = style.split(':')
+      .map((item) => item.trim());
+
+    if (property.length > 0) {
+      Object.assign(acc, { [property]: value });
```

Comment:
Using `Object.assign` to add a single property to an object is less efficient and more verbose than simply assigning the value directly with `acc[property] = value;`. This will improve both performance and readability.